composer.json

```diff
@@ -94,7 +94,8 @@
             "src/Tempest/EventBus/src/functions.php",
             "src/Tempest/Http/src/functions.php",
             "src/Tempest/Mapper/src/functions.php",
-            "src/Tempest/Reflection/src/functions.php"
+            "src/Tempest/Reflection/src/functions.php",
+            "src/Tempest/Support/src/functions.php"
```

Comment:
reduce the diff and overal convention (as far as I know)
```suggestion
            "src/Tempest/Support/src/functions.php",
```

Reply:
Trailing commas aren't allowed in JSON :(

Reply:
Oh right 🤦